### ISSUE TITLE
Nulllable warnings do not show as errors with WarningsAsErrors

### ISSUE THREAD

**Issue by MatisseHack** [OmniSharp/omnisharp-roslyn#2292]
I am trying to use `<WarningsAsErrors>nullable</WarningsAsErrors>` in a project, but nullable warnings are still not showing as errors. This is a simple project file that exhibits the issue:

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>net6.0</TargetFramework>
    <Nullable>enable</Nullable>
    <WarningsAsErrors>nullable</WarningsAsErrors>
  </PropertyGroup>
</Project>
```

The warnings are properly treated as errors when running `dotnet build`, but don't work in the editor. These symptoms seem similar to dotnet/roslyn#38371

**Comment by furai** [comment]
Did this stop working in recent versions or do I need some extra config to make it work? It's not working for me currently.

**Comment by JoeRobich** [comment]
@furai Please create a new one.
